### DISCUSSION TITLE
Welcome to Community engagement and informational resources discussion

### DISCUSSION THREAD

**Discussion by DGBNOOB** [DigiByte-Core/digibyte#248] [General]
“DigiByte Improvement Proposal (DIP) 7 aims to stop promoting third-party social networking apps, websites, and advocacy groups on the DigiByte.org website. By removing these external links, the goal is to concentrate community engagement and informational resources within the DigiByte-Core GitHub ecosystem”.

I am creating this discussion so that it can help in directing community engagement and help find informational resources within the DigiByte-Core GitHub ecosystem. This discussion may be a good start for someone who wants to contribute, make a comment, review current issues or Pull Request. The emphasis here is having a place to ask questions like "where can I help" and then being directed to where help is needed.

Currently there are a couple X post from DigiByteCoin that direct users to DigiByte-Core GitHub: (1) “If you’re a developer and wish to contribute to the protocol, please visit the DigiByte contribution guidelines and (2) For any technical updates/issues that you may want to discuss, please visit DigiByte-Core on GitHub”. 

The idea behind this discussion is that it’s a place where people can ask questions and then be directed to where they can find information or help contribute.

**Comment by Airith0** [comment]
Where can I find information on improvement proposals?

I can understand removing some links, and I can understand wanting to centralize communication efforts, but links are crucial for SEO. Diminishing your search potential hinders the chance for new people to organically find DigiByte.

**Reply by DGBNOOB** [reply]
Thank you for commenting, I’ll provide the information you’re inquiring about, please visit the discussion “dip-gto90-remove-links-from-website#7”, read and share your thoughts. 

To find information on DigiByte Improvement Proposals, there is a DIP repository in the DigiByte-Core GitHub Organization but I’ll share the link here [DigiByte-Core/dips](https://github.com/DigiByte-Core/dips).

If you’re looking for the [dip/discussion7](https://github.com/DigiByte-Core/dips/discussions/7) click on this link.

Keep in mind, the discussion is part of the dip process, this is the first proposal and it’s important that the process play out from start to finish.

**Reply by Airith0** [reply]
Thank you!

**Comment by timcbs** [comment]
Hello,
I am am trying to run a node on Apple Mac Mini, the instruction tell me to download the .DMG file from GitHub, I did that but, what I downloaded was a Zip File, I unzipped it and it made a bunch of folder with files.
Someone please help me, get the Digibyte-Core.DMG file.

Thank You in advance for Helping. Tim

**Reply by saltedlolly** [reply]
the current version is missing the binaries. For now, I suggest downloading v8.22.1

**Reply by timcbs** [reply]
Ok, I can download v8.22.1, but I am not sure how to do that, I am new to GitHub, I need a .dmg file, so I can drag it into the application folder, can you tell me how to do that.

Sorry if I am a pain in the ass. Tim

**Reply by saltedlolly** [reply]
You can find the DMG here: https://github.com/DigiByte-Core/digibyte/releases/tag/v8.22.1

**Comment by timcbs** [comment]
Thank You for the Help

Timothy Hill
***@***.***)

Parts-N-Things, Inc.
8245 Riverbanks Road Grants Pass, OR 97527
Phone 541-474-5965
Cell 541-441-3446

     On Monday, March 31, 2025 at 12:39:01 AM PDT, Olly Stedall ***@***.***> wrote:  
 
 


the current version is missing the binaries. For now, I suggest downloading v8.22.1
